README.rst

```diff
@@ -1,6 +1,220 @@
 Kinto python client
 ###################
 
-Python library to interact with a `Kinto <https://kinto.readthedocs.org>`_
-HTTP server.
+Kinto is a service allowing you to store and synchronize arbitrary data,
+attached to an user account. It's primary interface is HTTP.
+
+`kinto-client` is a python library which aims to ease the interaction with
+kinto for clients. `A project with related goals is
+also available for JavaScript <https://github.com/mozilla-services/cliquetis>`_.
+
+.. warning::
+
+    Everything described here is still pure fiction and no implementation
+    of this is ready to be used yet.
+
+
+Usage
+=====
+
+.. note::
+
+    The approach this library takes is different from the one described in the
+    `Cliquetis <https://github.com/mozilla-services/cliquetis>`_ javascript
+    client: Operations are always performed directly on the server, and no
+    synchronisation features are implemented yet.
+
+- The first version of this API doesn't cache any access nor provide any
+  refresh mechanism. If you want to be sure you have the last data available,
+  issue another call.
+
+Here are is an overview of what the API looks like:
+
+.. code-block:: python
+
+    from kintoclient import Bucket
+
+    bucket = Bucket('personal', server_url='http://localhost:8888/v1',
```

Comment:
Or maybe we want default by default: ``Bucket(server_url='http://localhost:8888/v1', auth=('alexis', 'p4ssw0rd'))``